ai_diffusion/workflow.py

```diff
@@ -626,6 +642,8 @@ def apply_edit_conditioning(
     input_latent: Output,
     control_layers: list[Control],
     vae: Output,
+    clip: Output,
```

Comment:
This is the only thing left that makes me somewhat uneasy. I think text encode should be done in the text encode function. It is fairly complex.

As it is, we would create a encode text prompt node, then abandon it and replace it with the qwen-text-encode node (leaving a dangling node around I guess). It might be better to pass the images to the existing `encode_text_prompt` function.

Reply:
I think that's ok now.